docs/kubernetes.md

```diff
@@ -62,7 +62,7 @@ Copy the [existing configuration](https://github.com/verdaccio/verdaccio/blob/ma
 and adapt it for your use case:
 
 ```bash
-wget https://github.com/verdaccio/verdaccio/blob/master/conf/full.yaml -O config.yaml
+wget https://raw.githubusercontent.com/verdaccio/verdaccio/master/conf/full.yaml -O config.yaml
```

Comment:
You need only change here, anything inside ` website/translated_docs/` is being handled by automation tools.  Please remove other changes.